.github/workflows/php.yml

```
@@ -0,0 +1,80 @@
+name: PHP tests
+on: [push, pull_request]
+jobs:
+    # Check there is no syntax errors in the project
+    php-linter:
+        name: PHP Syntax check 5.6|7.2|7.3
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.0.0
+
+            - name: PHP syntax checker 5.6
+              uses: prestashop/github-action-php-lint/5.6@master
+
+            - name: PHP syntax checker 7.2
+              uses: prestashop/github-action-php-lint/7.2@master
+
+            - name: PHP syntax checker 7.3
+              uses: prestashop/github-action-php-lint/7.3@master
+
+    # Check the PHP code follow the coding standards
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: '7.4'
+
+            - name: Checkout
+              uses: actions/checkout@v2.0.0
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                path: vendor
+                key: php-${{ hashFiles('composer.lock') }}
+
+            - name: Install dependencies
+              run: composer install
+
+            - name: Run PHP-CS-Fixer
+              run: ./vendor/bin/php-cs-fixer fix --dry-run --diff --using-cache=no --diff-format udiff
+
+    # Run PHPStan against the module and a PrestaShop release
+    phpstan:
+        name: PHPStan
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                presta-versions: ['1.7.1.0', 'latest']
```

Use more compatibility range:
```suggestion
                presta-versions: ['1.7.1.2', '1.7.2.5', '1.7.3.4', '1.7.4.4', '1.7.5.1', '1.7.6.9', 'latest']
```

---

Fixed